runtime/src/accounts_db.rs

```diff
@@ -1785,6 +1966,9 @@ impl AccountsDB {
                 }
                 all_removed_slot_storages.push(slot_removed_storages.clone());
             }
+            // It's possible that a slot is neither in the cache or storage, if the
+            // bank for that slot was on the wrong fork, and no account store has happened in
```

Comment:
I don't think no account store can happen. Bank::new_from_parent immediately creates some sysvars with the slot even if only seen ticks.

Reply:
Ah right, removed!